### ISSUE TITLE
Grid not initiated correctly with remote data

### ISSUE THREAD

**Issue by el95149** [donnelljenkins/aurelia-pagination#2]
Hello. I just installed datagrid 0.1.5 along with the latest navigation.
I have two different grids, which get populated inside activate() with remote data (via fetch).
Without pagination, the grid works perfectly.
When enabling pagination using, for example:
`<template replace-part="grid-footer-template">
                <td colspan.bind="columns.length">
                    <div>Total items: ${users.length}</div>
                    <pagination></pagination>
                </td>
            </template>
`

the grid is not populated. The code (Typescript) inside my VM for declaring and fetching the data is:

`export class CSpaceUsers extends BaseVM {
    users:any[] = [];
.........
    activate() {
        return this.http.fetch('user')
            .then(response => response.json())
            .then(users => {
                for(var user of users){
                    this.users.push(user);
                }
            });
    }
`
I also tried populating using: `this.users = users` instead of `push`.
Any help will be appreciated, thank you.

**Comment by donnelljenkins** [comment]
I do not know if this is the cause of your issue but while attempting to recreate it, I noticed that, with 2 grids on the same page, one grid's data was being replaced with the other's.  I fixed the data grid and released the fix (aurelia-datagrid v0.1.6).

After this, the pager worked with both grids (one pulling data from a remote site).

Try this new data grid release and see if it fixes this for you.
